File: nemiz-server/src/Nemiz/NemizBundle/Entity/Device.php
<?php
namespace Nemiz\NemizBundle\Entity;

use JMS\Serializer\Annotation\Type;
use Symfony\Component\Validator\Constraints as Assert;

class Device {
	/**
	 * @Type("string")
	 * @Assert\NotBlank(message = "device.name.not_blank")
	 * @Assert\Length(min=3, max=32, minMessage="device.name.min_length", maxMessage ="device.name.max_length")
	 */	
	private $name;
	
	/**
	 * @Type("string")
	 * @Assert\NotBlank(message = "device.type.not_blank")
	 */	
	private $type;
	
	/**
	 * @Type("string")
	 * @Assert\NotBlank(message = "device.token.not_blank")
	 */	
	private $token;
	
	public function getName() {
		return $this->name;
	}
	
	public function setName($name) {
		$this->name = $name;
		return $this;
	}
	
	public function getType() {
		return $this->type;
	}
	
	public function setType($type) {
		$this->type = $type;
		return $this;
	}
	
	public function getToken() {
		return $this->token;
	}
	
	public function setToken($token) {
		$this->token = $token;
		return $this;
	}
}

File: nemiz-android-app/app/src/main/java/co/nemiz/domain/Device.java
package co.nemiz.domain;

import java.io.Serializable;

public class Device implements Serializable {
    private static final long serialVersionUID = 7914723814094875345L;

    private String name, type, token;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}
